src/drivers/net/dwc_gmac/dwc_gmac.c

```diff
@@ -172,12 +175,12 @@ static int dwc_setup_txdesc(struct dwc_priv *priv, int idx, uint32_t buff,
 
 	assert(idx < TX_DESC_QUANTITY);
 
-	desc = &priv->txdesc_ring_paddr[idx];
-
-	desc->basic.des2 = buff;
+	desc = &priv->txdesc_ring[idx];
+	log_debug("txdesc_ring (%p): %d",  &((struct dma_extended_desc *)priv->txdesc_ring_dma)[idx], idx);
```

Comment:
Same for `&((struct dma_extended_desc *)priv->txdesc_ring_dma)[idx]` readability